src/NetAnalyzers/UnitTests/Microsoft.NetCore.Analyzers/Runtime/SpecifyCultureInfoTests.cs

```diff
@@ -652,14 +653,89 @@ End Sub
 End Class");
         }
 
-        private static DiagnosticResult GetCSharpResultAt(int line, int column, string arg1, string arg2, string arg3) =>
+        [Theory, WorkItem(3204, "https://github.com/dotnet/roslyn-analyzers/issues/3204")]
+        // Diagnostics
+        [InlineData("")]
+        [InlineData("dotnet_code_quality.CA1304.excluded_symbol_names = T:C")]
```

Comment:
Why does this setting trigger a violation?

Reply:
I have only handled method names in here: https://github.com/dotnet/roslyn-analyzers/pull/3325/files#diff-52e48dbabcedc53847defe0a973ebfb8R58